app/src/main/java/io/zenandroid/onlinego/views/BoardView.kt

```diff
@@ -23,10 +27,18 @@ class BoardView : View {
     // The logical size (in GO nodes) of the board.
     // Supported sizes: 9, 13 and 19
     //
+    var prefs = PreferenceManager.getDefaultSharedPreferences(OnlineGoApplication.instance.getBaseContext())
```

Comment:
As states above, board should be agnostic of shared preferences. It should instead have something like `var coordinatesVisible: Boolean = false` with setter